docs/guides/environmental-variables.md

```diff
@@ -0,0 +1,57 @@
+---
+title: CLI
+---
+
+Environmental variables are key-value pairs that are stored in our operating systems. Many of them come by default, and we can also create our own. They are used to store information that is used by our operating system or by other programs. For example, the `PATH` variable stores a list of directories that the operating system should look in when we run a command in the terminal. The `HOME` variable stores the path to our home directory.
```

Comment:
```suggestion
Environment variables are key-value pairs that are stored in our operating systems. Many of them come by default, and we can also create our own. They are used to store information that is used by our operating system or by other programs. For example, the `PATH` variable stores a list of directories that the operating system should look at when we run a command in the terminal. The `HOME` variable stores the path to our home directory.
```

Reply:
okay there are multiple occurences of "environmental", should i replace them all? also, the file name has "environmental"